base/src/main/java/org/aya/core/term/Term.java

```diff
@@ -137,6 +137,7 @@ default void traceExit(R r) {
     R visitFieldRef(@NotNull RefTerm.Field field, P p);
     R visitError(@NotNull ErrorTerm error, P p);
     R visitMetaPat(@NotNull RefTerm.MetaPat metaPat, P p);
+    R visitIntervalRef(@NotNull FormTerm.Interval interval, P p);
```

Comment:
```suggestion
    R visitInterval(@NotNull FormTerm.Interval interval, P p);
```

Reply:
I forgot to rename these methods after refactoring.